steps/repositories-describe/step.py

```diff
@@ -25,13 +30,28 @@
 except:
   direction = "asc"
 
+try:
+  fields = relay.get(D.fields)
+except:
+  fields = []
+
 gh = Github(token)
 
 try:
-  repos = gh.get_organization(org).get_repos(type=scope, sort=sort, direction=direction)
-  repos = list(map(lambda x: x.raw_data, repos))
+  if name:
+    repo = gh.get_organization(org).get_repo(name)
+    relay.outputs.set("repository", repo.raw_data)
```

Comment:
For consistency with following steps, maybe you'd want to make this e.g.

```python
relay.outputs.set("repositories", [repo.raw_data])
```

You're also not applying the fields filter in this case; you could simply set `repos = [repo.raw_data]` and pull most of the code from the `else:` up to make that easier.

Reply:
my thought was that if I were calling code explicitly to get a single result, that I'd want a singular response back, not a list of size one.